common/src/main/java/com/wynntils/core/components/Models.java

```diff
@@ -69,6 +68,7 @@ public final class Models {
     public static final AbilityTreeModel AbilityTree = new AbilityTreeModel();
     public static final BankModel Bank = new BankModel();
     public static final BeaconModel Beacon = new BeaconModel();
+    public static final ActivityModel Activity = new ActivityModel(Beacon);
```

Comment:
If ActivityModel constructor depends on Beacon, it should be moved under the (currently removed) heading "// Models with constructor dependencies, ordered alphabetically as far as possible" below.

Reply:
Done.